### PR TITLE
Update openapi.yaml

### PR DESCRIPTION
**Pull request by hoavoh**
add status for a sender address as an own number

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -1906,11 +1905,10 @@ paths:
                     account_id: my_account,
                     created_date: 2023-08-14T04:21:55.958Z,
                     last_modified_date: 2023-08-14T04:21:55.958Z,
-                    expiry: 2024-08-03T04:21:55.958Z
                   - id: 4a9cb0f4-f383-40b5-84dc-bbb6a3b210dd
                     sender_address: EXAMPLE3
```
Comment (arcdyc-sinch) [inline spec/openapi.yaml]: Please change to `+61491570156`

Comment (hoavoh) [reply spec/openapi.yaml]: I would change it, is it just to look more explicit?

Comment (david-porter-messagemedia) [reply spec/openapi.yaml]: it's because it should be an international phone number, not an alpha tag 👍

Comment (hoavoh) [reply spec/openapi.yaml]: yeah, agree.

Comment (hoavoh) [reply spec/openapi.yaml]: updated

**Review by arcdyc-sinch — APPROVED** [review]

**Review by david-porter-messagemedia — APPROVED** [review]

**Pull request merged**